x-pack/plugins/security_solution/public/entity_analytics/pages/entity_analytics_management_page.tsx

```diff
@@ -5,38 +5,174 @@
  * 2.0.
  */
 
-import React from 'react';
-import { EuiBetaBadge, EuiFlexGroup, EuiFlexItem, EuiPageHeader, EuiSpacer } from '@elastic/eui';
+import React, { useState, useEffect, useCallback } from 'react';
+import {
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiPageHeader,
+  EuiHorizontalRule,
+  EuiButton,
+  useEuiTheme,
+} from '@elastic/eui';
 
 import { RiskScorePreviewSection } from '../components/risk_score_preview_section';
 import { RiskScoreEnableSection } from '../components/risk_score_enable_section';
 import { ENTITY_ANALYTICS_RISK_SCORE } from '../../app/translations';
-import { BETA } from '../../common/translations';
 import { RiskEnginePrivilegesCallOut } from '../components/risk_engine_privileges_callout';
 import { useMissingRiskEnginePrivileges } from '../hooks/use_missing_risk_engine_privileges';
+import { RiskScoreUsefulLinksSection } from '../components/risk_score_useful_links_section';
+import { IncludeClosedAlertsSection } from '../components/include_closed_alerts_section';
+import { useRiskEngineStatus } from '../api/hooks/use_risk_engine_status';
+import { useScheduleNowRiskEngineMutation } from '../api/hooks/use_schedule_now_risk_engine_mutation';
+import { useAppToasts } from '../../common/hooks/use_app_toasts';
+import * as i18n from '../translations';
 
 export const EntityAnalyticsManagementPage = () => {
   const privileges = useMissingRiskEnginePrivileges();
+  const [includeClosedAlerts, setIncludeClosedAlerts] = useState(false);
+  const [from, setFrom] = useState(localStorage.getItem('dateStart') || 'now-30m');
+  const [to, setTo] = useState(localStorage.getItem('dateEnd') || 'now');
+  const { euiTheme } = useEuiTheme();
+  const { data: riskEngineStatus } = useRiskEngineStatus();
+  const currentRiskEngineStatus = riskEngineStatus?.risk_engine_status;
+  const runEngineEnabled = currentRiskEngineStatus === 'ENABLED';
+  const [isLoading, setIsLoading] = useState(false);
+  const { mutate: scheduleNowRiskEngine } = useScheduleNowRiskEngineMutation();
+  const { addSuccess } = useAppToasts();
+  const [nextRunTime, setNextRunTime] = useState('');
+
+  const handleRunEngineClick = async () => {
+    setIsLoading(true);
+    try {
+      await new Promise((resolve) => setTimeout(resolve, 500));
+      scheduleNowRiskEngine();
+    } catch (error) {
+      setIsLoading(false);
+    } finally {
+      setIsLoading(false);
+    }
+  };
+
+  const handleIncludeClosedAlertsToggle = (value: boolean) => {
+    setIncludeClosedAlerts(value);
+  };
+
+  const handleDateChange = ({ start, end }: { start: string; end: string }) => {
+    setFrom(start);
+    setTo(end);
+    localStorage.setItem('dateStart', start);
+    localStorage.setItem('dateEnd', end);
+  };
+
+  const calculateNextRunTime = useCallback((): string => {
```

Comment:
This logic is a duplication of https://github.com/elastic/kibana/blob/1de6fb5b639ccfe044bf388cb301e99c19890f03/x-pack/plugins/security_solution/public/entity_analytics/components/asset_criticality_file_uploader/components/schedule_risk_engine_callout.tsx#L63

Please reuse the logic. Create a reusable hook or component if you need to.

Reply:
Re-used the same logic. Thanks